src/components/control-component-factory.js

```diff
@@ -388,6 +382,13 @@ function createControlClass(s) {
       return this.props.getValue(event, this.props);
     }
 
+    /* eslint-disable camelcase */
```

Comment:
After getting CI to run properly, I saw that the linter was failing, so this fixes those issues:

https://travis-ci.org/github/davidkpiano/react-redux-form/jobs/666757092#L2124